sdk/builder_test.go

```diff
@@ -36,7 +36,7 @@ var _ = Describe("Builder", func() {
 		// set a cluster app override where the pre-release is different
 		const preReleasePrefix = "test"
 		releasesBuilder = releasesBuilder.
-			WithClusterApp("0.76.1-efdaba18e6e9866d62f3214f3d898b21c21e8b48", "").
+			WithClusterApp("1.0.0-efdaba18e6e9866d62f3214f3d898b21c21e8b48", "").
```

Comment:
Updating the test here and in other places below, as now in unit tests the latest release is `v25.0.0` (and it used to be `v25.1.0-demo.0`), so cluster-aws version is different in the base release.